docs/en/status-communities/about-community-request-approval.md

```diff
@@ -0,0 +1,24 @@
+---
+id: 514
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# About community request approval <!--include a plural on request or approval? ASk Emma-->
+
+When a community requires [manual approval][link-here] for membership requests, the community owner reviews each request and decides based on their criteria. If you're a community owner and want to configure manual approval, read [set up your community's approval options][link to the article].
```

Comment:
- 'the community owner <...> makes a decision'
- 'configure' -> 'set up'
- 'read' -> 'check out'

Reply:
Hey @Fabiomorais87 , @shamisem -- About the "read" vs. "check out".

This is a "read". If you're a community owner and want to configure manual approval, you must read the link; the rest of the article doesn't apply to you. Therefore, the link is not an optional "check this", it's a must "read" this.

I hope it makes sense!